SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -385,6 +397,16 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 	return ComponentDatas;
 }
 
+void EntityFactory::CreatePopulateSkeletonComponents(USpatialActorChannel& Channel, TArray<FWorkerComponentData>& OutComponentCreates,
+													 TArray<FWorkerComponentUpdate>& OutComponentUpdates, uint32& OutBytesWritten)
+{
+	TArray<FWorkerComponentData>& ComponentDatas = OutComponentCreates;
```

Comment:
Why not just use OutComponentCreates itself?

Reply:
Legacy, cleaning up - thanks for noticing